apollo-router/src/graphql/request.rs

```diff
@@ -211,20 +210,20 @@ impl Request {
                 mode = BatchingMode::BatchHttpLink.to_string() // Only supported mode right now
             );
             for entry in entries {
-                let bytes = serde_json::to_vec(entry)?;
+                let bytes = serde_json::to_vec(&entry)?;
                 result.push(Request::deserialize_from_bytes(&bytes.into())?);
             }
         } else {
-            let bytes = serde_json::to_vec(value)?;
+            let bytes = serde_json::to_vec(&value)?;
             result.push(Request::deserialize_from_bytes(&bytes.into())?);
         }
         Ok(result)
     }
 
-    fn process_query_values(value: &serde_json::Value) -> Result<Vec<Request>, serde_json::Error> {
-        let mut result = Request::allocate_result_array(value);
+    fn process_query_values(value: Value) -> Result<Vec<Request>, serde_json::Error> {
```

Comment:
I'm curious to know why you changed this function to take ownership of value here instead of ref as previously ?

Reply:
was part of serde_json => serde_json_bytes switch, but I will test if works with reference